alphastats/DataSet_Preprocess.py

```diff
@@ -183,29 +183,41 @@ def _linear_normalization(self, dataframe: pd.DataFrame):
 
     @ignore_warning(UserWarning)
     @ignore_warning(RuntimeWarning)
-    def _normalization(self, method: str):
+    def _normalization(self, method: str) -> None:
+        """Normalize across samples."""
+        # TODO make both sample and protein normalization available
         if method == "zscore":
             scaler = sklearn.preprocessing.StandardScaler()
+            # normalize samples => for preprocessing
             normalized_array = scaler.fit_transform(
                 self.mat.values.transpose()
             ).transpose()
+            # normalize proteins => for downstream processing
```

Comment:
cf. https://github.com/orgs/MannLabs/projects/12/views/1?filterQuery=-last-updated%3A14days&pane=issue&itemId=79587023

Reply:
Well done for resolving this! We can flesh out the details of what we will implement in the ticket.